src/angular-vs-repeat.js

```diff
@@ -255,12 +267,22 @@
                             $afterContent.css('width', '100%');
                         }
 
+                        var rafReinitId;
+                        function rafReinit(){
+                            if (reinitInRAF) {
+                                window.cancelAnimationFrame(rafReinitId);
```

Comment:
Is there a difference between canceling an existing and scheduling a new one or just letting the existing one finish and not queuing a new one?

Reply:
Yes, as reinitialized is called multiple times, our aim is to:
1) Reduce multiple reinitialiazation calls.
2) Reinitialize on the most recent data and attributes. (eg: the array that this lib is working on might change between multiple calls. eg from [/*empty*/] -> [1,2,3], so we want to reinit on the most recent data only)

Reply:
Thanks, I have updated the code.